api/portainer.go

```diff
@@ -150,9 +150,12 @@ type (
 		ID              RegistryID `json:"Id"`
 		Name            string     `json:"Name"`
 		URL             string     `json:"URL"`
+		Protocol        string     `json:"Protocol"`
+		Version         string     `json:"Version"`
```

Comment:
I don't see the point of storing the Version right now, is it used somewhere?

Reply:
No but I think we can use it client side to enable/disable the feature, as I think we only support browsing of v2 repository

Reply:
OK, good point.